lib/internal/readline.js

```diff
@@ -7,23 +7,38 @@
 const ansi =
   /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g;
 
+const kCSI = '\x1b';
```

Comment:
The CSI would be `\x1b[` (2 chars) or `\x9b`, this is just an escape character, so it would probably be a good idea to rename this to `kEscape`, or use `\x1b[` here